### ISSUE TITLE
ImportError: cannot import name 'SumVariationalStrategy'

### ISSUE THREAD

**Issue by Pinletrot** [jrg365/gpytorch#58]
Here's a minimal test code. 

```Python
import torch

from gpytorch.random_variables import GaussianRandomVariable
from torch.autograd import Variable

a = GaussianRandomVariable(Variable(torch.rand(3)), Variable(torch.rand(3, 3)))
b = GaussianRandomVariable(Variable(torch.rand(3)), Variable(torch.rand(3, 3)))

print(a + b)
```
The output is
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "~/packages/anaconda3/lib/python3.6/site-packages/gpytorch/random_variables/gaussian_random_variable.py", line 56, in __add__
>     from ..variational import SumVariationalStrategy
> ImportError: cannot import name 'SumVariationalStrategy'

---

The reason may be in the method "gpytorch.random_variables.gaussian_random_variable.__add__", where in line 56, it tries to 

```Python
from ..variational import SumVariationalStrategy
```

However, in "gpytorch/gpytorch/variational/__init__.py", there's no SumVariationalStrategy being imported.

```Python
from .variational_strategy import VariationalStrategy
from .mvn_variational_strategy import MVNVariationalStrategy


__all__ = [
    VariationalStrategy,
    MVNVariationalStrategy,
]
```

**Comment by jrg365** [comment]
Good catch, just pushed a quick fix to master.

**Comment by jrg365** [comment]
I'm actually reopening this -- we don't even have `SumVariationalStrategy` any more, so it is the code in `gaussian_random_variable.py` that needs fixing. Sorry about that -- the import fix seemed to work because I had an old .pyc file lying around.

**Comment by gpleiss** [comment]
Fixed by #59
